docs/user-guide/quickstart.md

```diff
@@ -1,5 +1,121 @@
+
+### Configure the onboard measurements
+
+VirtualShip is capable of taking underway temperature and salinity measurements, as well as onboard ADCP measurements, as the ship sails. To edit their configuration, open `ship_config.yaml`.
+
+Under `adcp_config` provide the configuration of your ADCP, so either `max_depth_meter: -1000.0` if you want to use the OceanObserver or `max_depth_meter: -150.0` if you want to use the SeaSeven (see [here](https://virtualship.readthedocs.io/en/latest/user-guide/assignments/Research_proposal_intro.html) for more details on the two ADCP types).
+
+If you don’t need onboard ADCP measurements, remove `adcp_config` and underlying lines from `ship_config.yaml`.
+
+If you do not want to collect temperature and salinity data, remove `ship_underwater_st_config` and underlying lines from `ship_config.yaml`.
+
+> [!NOTE] > **For advanced users only**: you can also edit the CTD, XBT, DRIFTER and ARGO_FLOAT configurations in `ship_config.yaml`. For CTD casts, the measurements will be taken to approximately 20 meters from the ocean floor by default, but this can be changed here if desired.
+
+## Fetch the data
+
+You are now ready to retrieve the input data required for simulating your virtual expedition from the [Copernicus Marine Data Store](https://data.marine.copernicus.eu/products). You will need to register for an account via https://data.marine.copernicus.eu/register.
+
+To retrieve the data, run the following command in your CLI:
 
-```{note}
-We don't yet have a quickstart guide. Please refer to the tutorials. If you would like to work on a quickstart guide, please [open an Issue](https://github.com/OceanParcels/virtualship/issues/new) to discuss.
 ```
+virtualship fetch EXPEDITION_NAME --username <USERNAME> --password <PASSWORD>
+```
+
+Replace `<USERNAME>` and `<PASSWORD>` with your own Copernicus Marine Data Store credentials. Alternatively, you can simply run `virtualship fetch EXPEDITION_NAME` and you will be prompted for your credentials instead.
+
+Waiting for your data download is a great time to practice your level of patience. A skill much needed in oceanographic fieldwork ;-)
+
+## Run the expedition
+
+Once your input data has finished downloading you can run your expedition using the command:
+
+```
+virtualship run EXPEDITION_NAME
+```
+
+Your command line output should look something like this...
+
+![GIF of example VirtualShip log output](example_log_instruments.gif)
+
+It might take up to an hour to simulate the measurements depending on your choices. Why not read up on some of the [onboard safety procedures](https://virtualship.readthedocs.io/en/latest/user-guide/assignments/Sail_the_ship.html#Emergency-procedures) or browse through [blogs and cruise reports](https://virtualship.readthedocs.io/en/latest/user-guide/assignments/Sail_the_ship.html#Reporting) in the meantime?!
```

Comment:
Not sure it's good practice to read the safety procedures when already doing the expedition ;-)